components/centraldashboard/cypress/e2e/landing-page.cy.ts

```diff
@@ -24,6 +24,26 @@ describe('Landing Page', () => {
     cy.get('main-page').shadow().find('landing-page').shadow().find('#MainCard > neon-animatable > div > #namespaceDisplay').find('span').should('exist').and('have.text', 'user-name');
   });
 
+  it('should email keep number', ()=>{
```

Comment:
Could this test title be a little bit more in english to describe what its testing